src/launcher/standalone_launcher/package.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
Please go through this and check which dependencies are actually used by launcher. We won't need everything here.